README.md

```diff
@@ -361,6 +361,31 @@ Optional open3d visualization of the scene:
 python -m stretch.app.read_map -i ~/Downloads/stretch\ output\ 2024-03-21/stretch_output_2024-03-21_13-44-19.pkl  --show-svm
 ```
 
+You can visualize instances in the voxel map with the `--show-instances` flag:
+
+```bash
+python -m stretch.app.read_map -i ~/Downloads/stretch\ output\ 2024-03-21/stretch_output_2024-03-21_13-44-19.pkl  --show-instances
```

Comment:
Is this file relevant? Or can it be hq_small.pkl?

Reply:
When running `python -m stretch.app.read_map -i hq_small.pkl --test-planning --start 4.5,1.3,2.1`, I'm getting an unpickling error: `_pickle.UnpicklingError: invalid load key, 'v'.`

I might need to install something related to git lfs - did you have to do this?
https://stackoverflow.com/questions/33049688/what-causes-the-error-pickle-unpicklingerror-invalid-load-key

Reply:
Yes, you need to install `git-lfs`, set it up, and re-clone `stretch_ai`

Reply:
updated readme + install, thanks for the feedback